Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/FormsTokenizationViewModel/FormPaymentMethodTokenizationViewModel+FormViews.swift

```diff
@@ -0,0 +1,219 @@
+//
+//  FormPaymentMethodTokenizationViewModel+FormViews.swift
+//  PrimerSDK
+//
+//  Copyright © 2022 Primer API ltd. All rights reserved.
+//
+
+#if canImport(UIKit)
+
+import Foundation
+
+
+extension FormPaymentMethodTokenizationViewModel {
+
+    // MARK: Input view
+    
+    func makeInputViews() -> [Input] {
+        
+        guard inputPaymentMethodTypes.contains(self.config.type) else { return [] }
+        
+        switch self.config.type {
```

Comment:
Can't fast be another `Input` and returned here?

Reply:
I thought about that. However, Fast via Rapyd APM doesn't explicitly require input but a different UX flow.
That's the main reason I decided to split the concepts.